azure-pipelines/build/linux/du/native/dopapt-ubuntu1804.yml

```diff
@@ -4,6 +4,8 @@
 variables:
 - name: imageVersion
   value: 0.8.0
+- name: containerImageVersion
```

Comment:
>containerImageVersion [](http://example.com/codeflow?start=8&length=21)

This isn't needed here .. but doesn't hurt either. How about adding a comment that it can be used in future if needed?